weave/trace_server/trace_server_interface.py

```diff
@@ -20,37 +75,45 @@ class CallSchema(BaseModel):
     # Optional display name of the call
     display_name: typing.Optional[str] = None
 
-    ## Trace ID
+    # Trace ID
     trace_id: str
-    ## Parent ID is optional because the call may be a root
+    # Parent ID is optional because the call may be a root
     parent_id: typing.Optional[str] = None
 
-    ## Start time is required
+    # Start time is required
     started_at: datetime.datetime
-    ## Attributes: properties of the call
-    attributes: typing.Dict[str, typing.Any]
+    # Attributes: properties of the call
+    attributes: AttributeMap
 
-    ## Inputs
+    # Inputs
     inputs: typing.Dict[str, typing.Any]
 
-    ## End time is required if finished
+    # End time is required if finished
     ended_at: typing.Optional[datetime.datetime] = None
 
-    ## Exception is present if the call failed
+    # Exception is present if the call failed
     exception: typing.Optional[str] = None
 
-    ## Outputs
+    # Outputs
     output: typing.Optional[typing.Any] = None
 
-    ## Summary: a summary of the call
-    summary: typing.Optional[typing.Dict[str, typing.Any]] = None
+    # Summary: a summary of the call
+    summary: typing.Optional[SummaryMap] = None
 
     # WB Metadata
     wb_user_id: typing.Optional[str] = None
     wb_run_id: typing.Optional[str] = None
 
     deleted_at: typing.Optional[datetime.datetime] = None
 
+    @field_serializer("attributes")
+    def serialize_attributes(self, v):
+        return dict(v)
+
+    @field_serializer("summary")
+    def serialize_summary(self, v):
+        return dict(v) if v else None
+
```

Comment:
...

Reply:
@tssweeney FYI, this _works_ but is a bit sus, how do you feel about this?

Reply:
any way you can avoid sending a load of Nones over the wire?

Reply:
Yep, experimenting with this now, setting `total=False` (like for the usage dict) @tssweeney